requirements.txt

```diff
@@ -1,7 +1,8 @@
-pytorch-lightning==1.2.0rc0  # todo: we shall align with real 1.2
+pytorch-lightning==1.2.3
```

Comment:
@tchaton `VALIDATING` stage is not part of `RunningStage` Enum in pl v1.2.3.

Reply:
same here

Reply:
and https://github.com/PyTorchLightning/lightning-flash/pull/133/files